From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 16 Sep 2017 20:09:30 +0200
Subject: Fix manpage generation

A footnote without a new line before seems to cause bad man page
generation by sphinx and trigger the following lintian warning:
W: streamlink: manpage-has-errors-from-man usr/share/man/man1/streamlink.1.gz 533: warning: macro `.' not defined

Forwarded: no
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/cli.rst | 1 +
 1 file changed, 1 insertion(+)

--- a/docs/cli.rst
+++ b/docs/cli.rst
@@ -340,6 +340,7 @@
 Both HTTP and SOCKS5 proxies are supported, authentication is supported for both types.
 
 For example:
+
 .. code-block:: console
 
     $ streamlink --http-proxy "http://user:pass@10.10.1.10:3128/" --https-proxy "socks5://10.10.1.10:1242"
